src/ds/d500/d500-options.h

```diff
@@ -0,0 +1,74 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#include "ds/ds-private.h"
+#include "core/options.h"
+#include "option.h"
+
+#pragma once
+
+namespace librealsense
+{
+    class rgb_tnr_option : public option
```

Comment:
@maloel make sure it's relevant for this project

Reply:
same

Reply:
Noted.